packages/terra-application-utility/src/utility/_UtilityMenu.jsx

```diff
@@ -328,7 +328,6 @@ class UtilityMenu extends React.Component {
         icon={<IconClose className={iconCloseClassNames} />}
         isCompact
         isIconOnly
-        aria-describedby="utility-menu-header"
```

Comment:
UX +1 , as a step 1 - this will help resolve the concern that was logged that _"this incorrectly conveys that Settings is the value or state of the Close User Options button. (WCAG 4.1.2: Name, Role, Value)"_.

Since the aria-describedby with this use creates the unclear context due the information and the order being read to the user, with the heading title coming immediately after button announcement, currently read to the user as "Close, button, User Settings", "Close, button, {h2 heading title}", etc.

As a step 2, we can test further improving with a followup PR by changing the actual label to include the heading title, e.g. "Close: {h2 heading title}", so that context order would be read as "Close: User Settings, button" instead.